src/app/components/layout/navbar/navbar.component.html

```diff
@@ -0,0 +1,38 @@
+<nav class="color border-gray-200 px-2 mx-auto">
+    <div class="container mx-auto flex flex-wrap items-center justify-between">
+
+
+        <button (click)="toggleMenu()" class="md:hidden ml-3 text-white" aria-controls="mobile-menu" [attr.aria-expanded]="isMenuOpen">
+            <span class="sr-only">Open main menu</span>
+            <svg class="w-6 h-6" fill="currentColor" viewBox="0 0 20 20" xmlns="http://www.w3.org/2000/svg" *ngIf="!isMenuOpen">
+                <path fill-rule="evenodd" d="M3 5a1 1 0 011-1h12a1 1 0 110 2H4a1 1 0 01-1-1zM3 10a1 1 0 011-1h12a1 1 0 110 2H4a1 1 0 01-1-1zM3 15a1 1 0 011-1h12a1 1 0 110 2H4a1 1 0 01-1-1z" clip-rule="evenodd"></path>
+            </svg>
+            <svg class="w-6 h-6" fill="currentColor" viewBox="0 0 20 20" xmlns="http://www.w3.org/2000/svg" *ngIf="isMenuOpen">
+                <path fill-rule="evenodd" d="M4.293 4.293a1 1 0 011.414 0L10 8.586l4.293-4.293a1 1 0 111.414 1.414L11.414 10l4.293 4.293a1 1 0 01-1.414 1.414L10 11.414l-4.293 4.293a1 1 0 01-1.414-1.414L8.586 10 4.293 5.707a1 1 0 010-1.414z" clip-rule="evenodd"></path>
+            </svg>
+        </button>
+
+
+
+        <div [ngClass]="{'hidden': !isMenuOpen, 'block': isMenuOpen}" class="md:block w-full md:w-auto mx-auto" id="mobile-menu">
+            <ul class="flex-col md:flex-row flex justify-center md:space-x-8 mt-4 md:mt-0">
+                <li>
+                    <a  class="md:bg-transparent text-white" >Home</a>
+                </li>
+                <li>
+                    <a  class="md:bg-transparent text-white">Offres d’emploi</a>
```

Comment:
as it stands, the idea of the project is just a repository for companies info, but if you want to add job offers ... you can work on it and make a MVP with one of the UI's and then we'll integrate it

for now, i suggest you remove this component until one of the features is ready